en/middleware/micrortps.md

```diff
@@ -1,24 +1,26 @@
 # RTPS/ROS2 Interface: PX4-FastRTPS Bridge
 
-The *PX4-FastRTPS Bridge* adds a Real Time Publish Subscribe (RTPS) interface to PX4, enabling the exchange of [uORB messages](../middleware/uorb.md) between PX4 components and (offboard) *FastRTPS* applications.
+The *PX4-FastRTPS Bridge* adds a Real Time Publish Subscribe (RTPS) interface to PX4, enabling the exchange of [uORB messages](../middleware/uorb.md) between PX4 components and (offboard) *Fast-RTPS* applications, including ones built over the ROS2/ROS frameworks.
 
-> **Note** RTPS is the underlying protocol of the Object Management Group's (OMG) Data Distribution Service (DDS) standard. It aims to enable scalable, real-time, dependable, high-performance and interoperable data communication using the publish/subscribe pattern. *FastRTPS* is a very lightweight cross-platform implementation of the latest version of the RTPS protocol and a minimum DDS API.
+> **Note** RTPS is the underlying protocol of the Object Management Group's (OMG) Data Distribution Service (DDS) standard. It aims to enable scalable, real-time, dependable, high-performance and interoperable data communication using the publish/subscribe pattern. *Fast-RTPS* is a very lightweight cross-platform implementation of the latest version of the RTPS protocol and a minimum DDS API.
 
 RTPS has been adopted as the middleware for the ROS2 (Robot Operating System). The bridge allows us to better integrate with ROS2, making it easy to share sensor values, commands, and other vehicle information.
 
-This topic describes the bridge architecture, how it is compiled, and how to write a simple FastRTSP application to subscribe to PX4 changes.
+This topic describes the bridge architecture, how it is compiled, and how to:
+1. Write a simple FastRTSP application to subscribe to PX4 changes;
+2. Or, if your application is on the ROS2/ROS1 side, understand how the `px4_ros_com` package works and how to use it to bridge your ROS nodes with PX4.
```

Comment:
FYI Markdown allows you to use `1.` for every bullet. That is useful if you might later move bullets around.

Suggest we change this to:
```
1. For ROS2/ROS1 applications, understand how the `px4_ros_com` package works and how to use it to bridge ROS nodes with PX4.
```

Reply:
Ok thanks